examples/src/main/java/com/jagrosh/jdautilities/examples/command/AboutCommand.java

```diff
@@ -15,107 +15,205 @@
  */
 package com.jagrosh.jdautilities.examples.command;
 
-import com.jagrosh.jdautilities.commons.JDAUtilitiesInfo;
 import com.jagrosh.jdautilities.command.Command;
+import com.jagrosh.jdautilities.command.CommandClient;
 import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jdautilities.commons.JDAUtilitiesInfo;
 import com.jagrosh.jdautilities.doc.standard.CommandInfo;
 import com.jagrosh.jdautilities.examples.doc.Author;
-import net.dv8tion.jda.bot.entities.ApplicationInfo;
 import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.JDA;
 import net.dv8tion.jda.core.JDAInfo;
 import net.dv8tion.jda.core.Permission;
+import net.dv8tion.jda.core.entities.Guild;
+import net.dv8tion.jda.core.entities.User;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.awt.*;
+import java.util.List;
 
 /**
+ * Sends generic information about the bot
  *
  * @author John Grosh (jagrosh)
  */
 @CommandInfo(
     name = "About",
     description = "Gets information about the bot."
 )
-@Author("John Grosh (jagrosh)")
-public class AboutCommand extends Command {
-    private boolean IS_AUTHOR = true;
-    private String REPLACEMENT_ICON = "+";
+@Author(" John Grosh (jagrosh)")
+public class AboutCommand extends Command
+{
+
     private final Color color;
     private final String description;
-    private final Permission[] perms;
-    private String oauthLink;
-    private final String[] features;
-    
-    public AboutCommand(Color color, String description, String[] features, Permission... perms)
+    private final Command[] commands;
```

Comment:
This shouldn't be stored here, as `event.getClient().getCommands()` will provide the currently-available command list.